instrumentation/jmx-metrics/javaagent/camel.md

```diff
@@ -0,0 +1,54 @@
+# Camel Metrics
+
+Here is the list of metrics based on MBeans exposed by Camel.
+
+| Metric Name                                            | Type    | Attributes                             | Description                                                                          |
+|--------------------------------------------------------|---------|----------------------------------------|--------------------------------------------------------------------------------------|
+| org.apache.camel.context.exchanges.completed           | Counter | context, camelVersion                  | Exchanges Completed grouped by Camel Context                                         |
```

Comment:
Perhaps use `camel` instead of `org.apache.camel`, this seems to better align with existing JMX metrics. @PeterF778 do you have any suggestions? Also I'd use `exchange` instead of `exchanges`.